dotCMS/src/main/java/com/dotcms/rest/api/v1/event/SystemEventsDelegate.java

```diff
@@ -78,8 +80,6 @@ public void execute(final AppContext context) {
                 HibernateUtil.closeSession();
```

Comment:
shouldn't be removed this one HibernateUtil.closeSession(); too?

Reply:
yeah removed entire `finally`